activiti-intellij-plugin/src/main/resources/xsds/activiti-6.0.xsd

```diff
@@ -332,7 +332,7 @@
         </annotation>
     </attribute>
 
-    <attribute name="candidateUsers">
```

Comment:
Please remove all modifications in XSD files - they came from official sources and it is better not to touch them as they are schema definition